.mergify.yml

```diff
@@ -1,4 +1,11 @@
 pull_request_rules:
+  # if there is a conflict in a backport PR, ping the author to send a proper backport PR
+  - name: ping author on conflicts
+    conditions:
+      - conflict
+    actions:
+      comment:
+        message: This pull request is now in conflict, please @{{author}} rebase it.
```

Comment:
```suggestion
        message: This pull request has merge conflicts that must be resolved before it can be merged. @{{author}} please rebase it.
```

Reply:
How about a link to the dev guide that mentions rebasing?